src/plugins/cableCar.ts

```diff
@@ -94,7 +94,13 @@ export class CableCar {
     const fetchResponse = event.detail.fetchResponse
 
     if (fetchResponse == null) return
-    if ((fetchResponse.contentType?.match(this.mimeType)) == null) return
+
+    const contentTypeMatchesAccept = this.mimeType
+      .split(/, */)
+      .reduce((result: boolean, s: string) => {
+        return result || !((fetchResponse.contentType?.match(s)) == null)
+      }, false)
+    if (!contentTypeMatchesAccept) return
```

Comment:
I think we can just do:

```js
    if (fetchResponse.contentType == null) return
    if (!this.mimeType.includes(fetchResponse.contentType)) return
```

Reply:
Unfortunately I don't think that will work because the response content type sent by rails includes character encoding info that won't be present in this.mimeType